docs/user-guide/building-collection.md

```diff
@@ -0,0 +1,86 @@
+- [ansible-navigator](https://github.com/ansible/ansible-navigator)
+- [VS Code Ansible extension](https://github.com/ansible/vscode-ansible)
+
+## Scaffolding collection using ansible-creator
+
+- Open VS Code and click the Ansible icon in the activity bar to access Ansible Creator section. Click on "Get Started" under that section to open the menu page of Ansible Creator in VS Code.
+
+- Check system requirements and install ansible-creator if needed. Ensure all requirements in the `system requirements` box have green ticks.
+
+![ansible-creator-menu](../media/ansible-creator-menu-installed.png)
+
+- Click "Initialize a collection" to open the "Init" interface. Fill the form with the collection name, initialization path, verbosity, and logging options. Click "Create" for scaffolding the collection in the desired location. You have the open to review the logs or open the log file in VS Code editor for details.
+
+- Click on `Open collection` button to add collection folder to the workspace.
+
+![scaffold-collection](../media/create-collection.png)
+
+NOTE: For a more detailed explanation about using Ansible Creator in the VS Code Ansible Extension, refer to [doc: ansible-creator].
+
+## Installing collection using ansible-development-environment
+
+- With the initial collection structure in place, use 'ansible-development-environment' to install the newly created collection in editable mode, similar to Python modules.
+
+- Navigate to the collection directory and run:
+
+```console
+$ ansible-development-environment install -e .
+```
+
+- This installation method adds the collection in the system paths so that ansible knows about it. Additionally, it enhances the development process by allowing on-the-go changes to the module code.
+
+- You can check if the collection is installed or not by using ansible-galaxy command. In the terminal, running the following command should show the name of the newly created collection:
+
+```console
+$ ansible-galaxy collection list
+```
+
+## Add python code to bring the collection to life
+
+- Navigate to the collection directory and navigate to plugins/modules/. Add create a [module-name].py file and documentation, examples and logic to the module.
+
+- Due to its installation method using pip4a, you have the ability to change module code dynamically and observe the effects during playbook execution.
+
+NOTE: for details regarding the module development, refer to the [ansible module development docs](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html).
+
+## Using ansible-lint to check module syntax in the playbook
+
+- Leverage vscode-ansible extension and ansible-lint to gain insights into the collection without running the playbook
+
+- Create a simple playbook under the 'playbooks' directory.
+
+- Once you have completed writing the playbook that uses the newly collection module, saving the file will automatically run ansible-lint on the playbook.
```

Comment:
```suggestion
- When you have completed writing the playbook that uses the newly collection module, saving the file will automatically run ansible-lint on the playbook.
```
Technical writing standards advise to use "When" or "after" instead of "once", as they're a bit clearer, especially for translations.

Reply:
```suggestion
- Once you have completed writing the playbook that uses the newly collection module, saving the file will automatically run `ansible-lint` on the playbook.
```